react/features/base/util/openURLInBrowser.native.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { Linking } from 'react-native';
+
+/**
+ * Opens URL in the browser.
+ *
+ * @param {string} url - The URL to be opened.
+ * @returns {void}
+ */
+export function openURLInBrowser(url: string) {
```

Comment:
I like you created this function! Are you considering refactoring all other mobile links (there are not a lot of them) using this instead of invoking Linking.... individually?

Reply:
Good idea! Will do but maybe trough another PR! I don't want to block this one!